portfolio/.eslintrc

```diff
@@ -10,6 +10,7 @@
   "rules": {
     "require-jsdoc": "off",
     "valid-jsdoc": "off",
-    "object-curly-spacing": "off"
+    "object-curly-spacing": "off",
+    "max-len": "off"
```

Comment:
Is this the rule for the max character length of a line? Why turn this off?

Reply:
I recommended we turn it off after seeing that it would require us to break up long strings which imo is less readable than keeping a long string on one line.